### DISCUSSION TITLE
Question: When week 7 itch io release?

### DISCUSSION THREAD

**Discussion by DzGithub2805** [FunkinCrew/Funkin#1561] [General]
## What is your question?
Just asking

**Comment by suprstarrd** [comment]
what do you mean when is week 7? there's only 2 weeks. have you tried spooky yet. who's tankman

(okay but seriously it's probably staying newgrounds-exclusive for a while, give it a few more months-- i'm not a dev though so take that lightly)

**Comment by ghost** [comment]
Week 7 will release when:

1. They get cutscenes working on desktop
2. NG link is 1st or 2nd on Google search for "Friday Night Funkin'"

**Comment by SpringyToast** [comment]
ngl I don't really think the 2nd will make it because of the Youtube links but I gotta say Crazy games is directing people to NG through their basically rip-off FNF port since they don't have week 7 so they have a link to NG so does that count as 2nd?

**Comment by suprstarrd** [comment]
no, crazy games is literally one of the reasons they made that goal

**Comment by aflacc** [comment]
i hate it that people think its ok to just put week 7 on your site

**Comment by suprstarrd** [comment]
exactly. like no, Greg, you're not helping the developers, you're helping your mother do the dishes

**Comment by SpringyToast** [comment]
Crazygames is still pretty shitty but at least they actually had a little bit of respect for the Devs to not put week 7 in their repost of the game so they get a little bit of respect from me like a tiny grain of respect because they still just stole the source code

**Comment by ghost** [comment]
Yeah
